content/en/docs/kubernetes/operator/troubleshooting/automatic.md

```diff
@@ -0,0 +1,179 @@
+ Generation:          1
+ Resource Version:    3385
+ UID:                 646661d5-a8fc-4b64-80b7-8587c9865f53
+Spec:
+...
+ Exporter:
+   Endpoint:  http://otel-collector-collector.opentelemetry.svc.cluster.local:4318
+...
+ Propagators:
+   tracecontext
+   baggage
+ Python:
+   Image:  ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-python:0.39b0
+   Resource Requirements:
+     Limits:
+       Cpu:     500m
+       Memory:  32Mi
+     Requests:
+       Cpu:     50m
+       Memory:  32Mi
+ Resource:
+ Sampler:
+Events:  <none>
+```
+
+### Check the OpenTelemetry Operator Logs
+
+Check the OpenTelemetry Operator logs for errors, by running this command:
+
+```shell
+kubectl logs -l app.kubernetes.io/name=opentelemetry-operator --container manager -n opentelemetry-operator-system --follow
+```
+
+The logs should not show any errors related to auto-instrumentation errors.
+
+### Check deployment order
+
+Order matters. The `Instrumentation` resource must be deployed before deploying
+the corresponding `Deployment` resource(s) being auto-instrumented.
+
+Consider the following auto-instrumentation annotation snippet:
+
+```yaml
+annotations:
+  instrumentation.opentelemetry.io/inject-python: 'true'
+```
+
+It tells the OpenTelemetry Operator to look for an `Instrumentation` resource in
+the pod’s namespace. It also tells the Operator to inject Python
+auto-instrumentation into the pod.
+
+When the pod starts up, the annotation tells the Operator to look for an
+`Instrumentation` resource in the pod’s namespace, and to inject Python
+auto-instrumentation into the pod. It adds an
+[init-container](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/)
+to the application’s pod, called `opentelemetry-auto-instrumentation`, which is
+then used to injects the auto-instrumentation into the app container.
+
+But if the `Instrumentation` resource isn’t present by the time the `Deployment`
+is deployed, the `init-container` can’t be created. This means that if the
+`Deployment` resource is deployed _before_ you deploy the `Instrumentation`
+resource, the auto-instrumentation will fail to initialize.
+
+Check that the `opentelemetry-auto-instrumentation` `init-container` has started
+up correctly (or has even started up at all), by running the following command:
+
+```shell
+kubectl get events -n <your_app_namespace>
+```
+
+Which should result in output that looks something like this:
+
+```text
+53s         Normal   Created             pod/py-otel-server-7f54bf4cbc-p8wmj    Created container opentelemetry-auto-instrumentation
+53s         Normal   Started             pod/py-otel-server-7f54bf4cbc-p8wmj    Started container opentelemetry-auto-instrumentation
+```
+
+If the output is missing `Created` and/or `Started` entries for
+`opentelemetry-auto-instrumentation`, then it means that there is an issue with
+your auto-instrumentation configuration. This can be the result of any of the
+following:
```

Comment:
```suggestion
If the output is missing `Created` or `Started` entries for
`opentelemetry-auto-instrumentation`, there might be an issue with
your auto-instrumentation configuration. This can be the result of any of the
following:
```